ouroboros-consensus/src/Ouroboros/Storage/ImmutableDB/Index.hs

```diff
@@ -346,3 +350,22 @@ truncateToSlots slots index@(MkIndex offsets ebbHash)
   = index
   | otherwise
   = MkIndex (V.take (fromIntegral slots + 1) offsets) ebbHash
+
+{-------------------------------------------------------------------------------
+  Auxiliary: encoding and decoding the EBB hash
+
+  When no EBB is present we use an all-NULL bytestring.
```

Comment:
"all-NULL bytestring" is confusing, because `"\NUL\NUL\NUL"` is an all-NUL(L) bytestring, while you actually mean an "empty bytestring".

Reply:
Ah, yes, stale comment, good catch. Fix will fix in a separate PR, so that we can merge this and don't incur more CI delays.